### DISCUSSION TITLE
how process.stdout be set to the process object

### DISCUSSION THREAD

**Discussion by LinYUAN-code** [nodejs/help#4937] [Q&A] [answered]
how process.stdout be set to the process object in source Code

**Comment by aduh95** [comment, marked as answer]
You can find it here: https://github.com/nodejs/node/blob/c131fbb65230cc8f62cc0f41682388507a669684/lib/internal/bootstrap/switches/is_main_thread.js#L16-L26

**Reply by LinYUAN-code** [reply]
thank u so muck！
